main.py

```diff
@@ -70,14 +107,30 @@ async def on_message(self, message):
         if message.author.id in self.state["admins"]:
             if message.content == self.prefix + 'sub-hive':
                 channel = message.channel.id
-                self.state["channels"].append(channel)
+                self.state["hive_channels"].append(channel)
                 util.save_state(self.state)
                 await message.channel.send('Added channel to Portal Hive Status Updates')
                 return
             if message.content == self.prefix + 'unsub-hive':
                 channel = message.channel.id
                 try:
-                    self.state["channels"].remove(channel)
+                    self.state["hive_channels"].remove(channel)
+                    await message.channel.send('Removed channel to Portal Hive Status Updates')
+                except:
+                    await message.channel.send('Failed to remove channel to Portal Hive Status Updates')
+                util.save_state(self.state)
+                return
+            
+            if message.content == self.prefix + 'sub-glados':
+                channel = message.channel.id
+                self.state["glados_channels"].append(channel)
+                util.save_state(self.state)
+                await message.channel.send('Added channel to Portal Hive Status Updates')
+                return
+            if message.content == self.prefix + 'unsub-glados':
+                channel = message.channel.id
+                try:
+                    self.state["glados_channels"].remove(channel)
```

Comment:
same here it should be glados instead of portal hive
there are 2 `message.channel.send` below this line